@here/harp-mapview-decoder/test/TileGeometryLoaderTest.ts

```diff
@@ -230,14 +224,14 @@ describe("TileGeometryLoader", function() {
 
             expect(mapView.taskQueue.processNext(TileTaskGroups.CREATE)).equal(true);
 
-            geometryLoader.waitFinished().should.be.rejected.then(() => {
+            return geometryLoader.waitFinished().should.be.rejected.then(() => {
```

Comment:
this file mixes returning a promise & async / await, please choose one way and keep it consistent

Reply:
will do